service/entityresolution/keycloak/keycloak_entity_resolution.go

```diff
@@ -33,28 +34,17 @@ type KeyCloakConnector struct {
 }
 
 func EntityResolution(ctx context.Context,
-	req *authorization.IdpPluginRequest, config *authorization.IdpConfig) (*authorization.IdpPluginResponse, error) {
+	req *entityresolution.ResolveEntitiesRequest, kcConfig KeycloakConfig) (entityresolution.ResolveEntitiesResponse, error) {
 	// note this only logs when run in test not when running in the OPE engine.
-	slog.DebugContext(ctx, "EntityResolution", "req", fmt.Sprintf("%+v", req), "config", fmt.Sprintf("%+v", config))
-	jsonString, err := json.Marshal(config.GetConfig().AsMap())
+	slog.DebugContext(ctx, "EntityResolution", "req", fmt.Sprintf("%+v", req), "config", fmt.Sprintf("%+v", kcConfig))
```

Comment:
Is it a problem if we log this config when it contains the client id and secret?

Reply:
with the debug context it should only log when running tests, not when running in the OPA engine

Reply:
Even when log level is set to `debug` it won't log?

Reply:
ah no you're right, i forgot this was when it was an opa plugin, i moved the config logging to a separate log and commented it out